.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "webapp/src/main/webapp/themes/cu-boulder/expmap"]
 	path = webapp/src/main/webapp/themes/cu-boulder/expmap
 	url = https://github.com/cu-boulder/expertsmap
+[submodule "webapp/src/main/webapp/themes/cu-boulder/facetview-HTML"]
+	path = webapp/src/main/webapp/themes/cu-boulder/facetview-HTML
+	url = https://dottermann@github.com/cu-boulder/facetview-HTML.git
```

Comment:
What's this URL for? Should it be qualified with the username?

Reply:
nice catch. This is related to us not having our github private keys setup. It's been a while since I worked on the VIVO stuff with another team member. Nate and I used to run into these issues. Perhaps it's best that David and I just resolve this now before we merge.